README.md

```diff
@@ -18,6 +19,56 @@ Specifically, if it enables services to scale _during_ deployment.
 > [!TIP]
 > A dashboard combining CloudFormation events, ASG activity and ASG metrics is available here - https://metrics.gutools.co.uk/d/cdvsv1d6vhp1cb/testing-asg-rolling-update.
 
+## Summary
+Can `AutoScalingRollingUpdate` be used to deploy updates to an auto-scaling group? Yes.
+- If not using horizontal scaling, `AutoScalingRollingUpdate` can be used, with any configuration.
+- If using horizontal scaling, `AutoScalingRollingUpdate` with dynamic configuration, is a viable deployment strategy. However, deployment times are significantly increased in certain conditions. 
```

Comment:
> However, deployment times are significantly increased in certain conditions.

I think it might be worth reframing this slightly.

In the typical case (where we can replace all instances in a single batch) `AutoScalingRollingUpdate` deployments should be of a similar duration to Riff-Raff's `autoscaling` deployment type because it is essentially using the same strategy.

IIUC, longer deployment times occur when AWS is forced to replace the instances in more than 1 batch. This happens when `desired > max / 2` and we set `minimumInstancesInService` accordingly (e.g. if there are 6 instances running and max is 9, we need to replace the instances in two batches of 3 as we cannot add 6 instances at once or drop below 6 running instances).

Currently in this scenario Riff-Raff's `autoscaling` deployment will fail fast (because there is not enough headroom to double desired), so although the deployment times for `AutoScalingRollingUpdate` are increased by comparison, the fact that they _can_ succeed seems pretty crucial to me. Perhaps we want to phrase as something like:

> It can also deploy in scenarios that Riff-Raff currently fails to support, although these deployments will be considerably slower than a 'typical' deployment.

Reply:
Hopefully the latest (force pushed) version addresses this.